motor/providers/container/image/docker.go

```diff
@@ -35,7 +35,8 @@ func (r ShaReference) Scope(scope string) string {
 }
 
 func LoadImageFromDockerEngine(sha string) (v1.Image, io.ReadCloser, error) {
-	img, err := daemon.Image(&ShaReference{SHA: strings.Replace(sha, "sha256:", "", -1)})
+	opts := []daemon.Option{daemon.WithUnbufferedOpener()}
+	img, err := daemon.Image(&ShaReference{SHA: strings.Replace(sha, "sha256:", "", -1)}, opts...)
```

Comment:
By default, this is created with a bufferedReader: https://github.com/google/go-containerregistry/blob/main/pkg/v1/daemon/image.go#L62